src/utils/components/DiskModal/components/DiskSourceSelect/components/DiskSourceUrlInput/URLSourceHelperText.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+
+import { OS_IMAGE_LINKS, OS_NAME_TYPES } from '@kubevirt-utils/resources/template';
+
+import { urlSourceHelperTextMapper } from './utils/constants';
+import { DiskSourceUrlInputProps } from './utils/types';
+
+const URLSourceHelperText: FC<DiskSourceUrlInputProps> = ({ os }) => {
+  const { afterLabelText, beforeLabelText, label } = urlSourceHelperTextMapper[os];
+  return (
+    <>
+      {beforeLabelText}
+      <strong>
+        <a href={OS_IMAGE_LINKS[os || OS_NAME_TYPES.fedora]} rel="noreferrer" target="_blank">
```

Comment:
<Link to="" target="_blank" rel="noreferrer" />

Reply:
??